### ISSUE TITLE
Updating deployment won't update available pods 

### ISSUE THREAD

**Issue by adietish** [redhat-developer/intellij-kubernetes#306]
related to #286 

**WORKAROUND:**
1. EXEC: expand the category `Pods` (in tree)

The reason for this is that pods are not watched. They're as soon as you expand `Pods`. When changing the replicas in the deployment the pods for the deployment are changed (existing pods are terminated, new pods are created). The plugin won't notice this change if it is not watching the pods.

**Steps:**
1. ASSERT:  have a deployment
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: nginx-rolling
  labels:
    app: nginx-rolling
spec:
  replicas: 1
  selector:
    matchLabels:
      app: nginx-rolling
  strategy:
    type: RollingUpdate
    rollingUpdate:
      maxSurge: 1
      maxUnavailable: 0
  template:
    metadata:
      labels:
        app: nginx-rolling
    spec:
      containers:
        - image: nginx
          name: nginx
          ports:
            - containerPort: 8080
```
1. EXEC: open deployment in editor (double click deployment in resource tree)
1. ASSERT: **1** pod `nginx-rolling-xxxxxxx` is shown as child to the deployment `nginx-rolling`
1. EXEC: change `spec` > `replicas` to **2** & Push
1. ASSERT: deployment `nginx-rolling` is collapsed
1. EXEC: expand `nginx-rolling`

**Result:**
 deployment `nginx-rolling` still has **1** pod `nginx-rolling-xxxxxxx` as child (should have **2**) 
`Refresh` won't help.

**Comment by adietish** [comment]
This is fixed in the PR #292

**Comment by adietish** [comment]
#292 was merged, closing.
